chart/templates/deployment.yaml

```diff
@@ -0,0 +1,103 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
```

Comment:
It would be good to include an annotation to help trigger a deployment update when the run-config changes:

```
annotations:
  checksum/run-config: {{ include (print $.Template.BasePath "/config.yaml") . | sha256sum }}
```

https://helm.sh/docs/howto/charts_tips_and_tricks/#automatically-roll-deployments